templates/docs/examples/patterns/newsletter-signup/25-75.html

```diff
@@ -0,0 +1,83 @@
+{% extends "_layouts/examples.html" %}
+{% from '_macros/vf_newsletter-signup.jinja' import vf_newsletter_signup %}
+{% block title %}Newsletter signup{% endblock %}
+{% block standalone_css %}patterns_all{% endblock %}
+{% block content %}
+    {% call(slot) vf_newsletter_signup(layout="25-75", title_text="Newsletter signup", form_id="mktoForm_4960", return_url="https://ubuntu.com/#contact-form-success", input_label='Work email', checkbox_id='canonicalUpdatesOptIn', checkbox_label='I agree to receive information about Canonical\'s products and services.') %}
+        {%- if slot == 'description' -%}
+            <p>Get the latest Ubuntu news and updates in your inbox.</p>
+        {%- endif -%}
+        {%- if slot == 'hidden_fields' -%}
+            <input type="hidden"
+                   aria-hidden="true"
+                   aria-label="hidden field"
+                   name="returnURL"
+                   value="https://ubuntu.com/#contact-form-success" />
+            <input type="hidden"
```

Comment:
You should be able to remove these hidden field slots from your examples, now that they have a default in the macro implementation itself.